### PR TITLE
Changelog updates for 1.5.3 etc

### PR DESCRIPTION
**Pull request by mladlow**
Wanted to make a PR to check for comments

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -156,12 +168,24 @@ BUG FIXES:
 * ui: Disallow max versions value of large than 9999999999999999 on kv2 secrets engine. [[GH-9242](https://github.com/hashicorp/vault/pull/9242)]
 * ui: Add and upgrade missing dependencies to resolve a failure with `make static-dist`. [[GH-9277](https://github.com/hashicorp/vault/pull/9371)]
 
-## 1.4.6 (TBD)
+## 1.4.7 (TBD)
 
 IMPROVEMENTS:
 
 * secret/azure: Use write-ahead-logs to cleanup any orphaned Service Principals [[GH-9773](https://github.com/hashicorp/vault/pull/9773)]
```
Comment (calvn) [inline CHANGELOG.md:175]: This should be in on 1.5.3 though: https://github.com/hashicorp/vault/pull/9769/files

Comment (kalafut) [reply CHANGELOG.md:175]: That feature debuted in 1.5.1

Comment (mladlow) [reply CHANGELOG.md:175]: Should I move it to 1.5.1 then?

Comment (calvn) [reply CHANGELOG.md:175]: Yep, I'd say so. Jim verified that this was in 1.5.1.

Comment (kalafut) [reply CHANGELOG.md:175]: Yes, it should be in the 1.5.1 section (and only 1.5.1)

**Pull request merged**